wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -137,21 +151,18 @@
           },
           "selector": {
             "$ref": "#/definitions/selector",
-            "description": "Describes the objects selector for the recipe type tools",
+            "description": "Describes the objects selector for the recipe type tools. Allows to pick-up only selected items from k8s/openshift list",
             "examples": [
-              {
-                "app.kubernetes.io/name": "mysql",
-                "app.kubernetes.io/component": "database",
-                "app.kubernetes.io/part-of": "petclinic"
-              }
+              "{\n   \"app.kubernetes.io/name\" : \"mysql\", \n   \"app.kubernetes.io/component\" : \"database\", \n   \"app.kubernetes.io/part-of\" : \"petclinic\" \n}"
```

Comment:
It was easy to read the previous variant.
Is updated way looks good in auto-generated doc?

Reply:
Yes, library is expect examples to be array of strings, not objects. And didn't render it otherways.